Gemfile

```diff
@@ -1,6 +1,5 @@
 source "https://rubygems.org"
 git_source(:github) { |repo| "https://github.com/#{repo}.git" }
-
```

Comment:
We should add these newlines back into the Gemfile. It doesn't make sense in the context of this pull request to remove them. We can remove them in another pull request if we want to do that. What do you think?

(ditto throughout)

Reply:
@purinkle I agree to adding the newlines back